conans/client/generators/text.py

```diff
@@ -65,4 +144,10 @@ def content(self):
                 all_flags = template.format(dep=dep, deps=deps, config=":" + config)
                 sections.append(all_flags)
 
+        # Generate the user info variables as [LIB_A_USER_VAR]\n
+        for dep, the_vars in self._deps_user_info.items():
+            for name, value in the_vars.vars.items():
+                header = "[USER_%s:%s]" % (dep, name)
```

Comment:
Would it make more sense a section per package, then name=value as a list inside that section?